### PR TITLE
{Yolanda} iP

### PR DESCRIPTION
**Pull request by yolanmz**

### CODE REVIEW

File: src/main/java/Duke.java [hunk 1]
```diff
@@ -6,5 +8,40 @@ public static void main(String[] args) {
...
                 + "|____/ \\__,_|_|\\_\\___|\n";
         System.out.println("Hello from\n" + logo);
+        System.out.println("------------------------------------------------");
+        System.out.println("Hello! I'm Duke");
+        System.out.println("What can I do for you?\n");
+        System.out.println("------------------------------------------------");
+
+        Scanner in = new Scanner(System.in);
+        String input = in.nextLine();
+        String[] list = new String[100];
```
Comment (DeetoMok) [inline src/main/java/Duke.java:18]: I think it would be nice to specify the list, e.g. "taskList"

File: src/main/java/Duke.java [hunk 2]
```diff
@@ -6,5 +8,40 @@ public static void main(String[] args) {
...
+                System.out.println("------------------------------------------------");
+                System.out.println("added: " + input);
+                System.out.println("------------------------------------------------");
+
+            }
+            in = new Scanner(System.in);
+            input = in.nextLine();
+
+        }
+        if(input.equals("bye")) {
```
Comment (DeetoMok) [inline src/main/java/Duke.java:40]: This if-check seems redundant, as once the while loop is broken, this condition will always be satisfied.

File: src/main/java/Duke.java [hunk 3]
```diff
@@ -1,3 +1,5 @@
+import java.util.Scanner;
```
Comment (DeetoMok) [inline src/main/java/Duke.java:1]: I like how you only import the packages you need!

**Review by DeetoMok — COMMENTED** [review]
Good Job 👍

File: src/main/java/Duke.java [hunk 4]
```diff
@@ -6,5 +8,40 @@ public static void main(String[] args) {
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
         System.out.println("Hello from\n" + logo);
+        System.out.println("------------------------------------------------");
```
Comment (DeetoMok) [inline src/main/java/Duke.java:11]: You can define your magic strings and magic numbers as a constant 😄

**Comment by damithc** [comment]
Closing this PR as the author has an iP exemption.

File: src/main/java/Duke.java (hunk 2 above)
Comment (yolanmz) [reply src/main/java/Duke.java:40]: I just noticed that. Thank you!

**Pull request closed without merging**